testsuite/features/support/commonlib.rb

```diff
@@ -195,6 +195,8 @@ def generate_repository_name(repo_url)
   repo_name.delete_prefix! 'http://download.suse.de/ibs/SUSE:/Maintenance:/'
   repo_name.delete_prefix! 'http://download.suse.de/download/ibs/SUSE:/Maintenance:/'
   repo_name.delete_prefix! 'http://download.suse.de/download/ibs/SUSE:/'
+  repo_name.sub!(/http:\/\/.*compute.internal\/SUSE:\//,'')
```

Comment:
To be consistent, we should using `sub!` or `delete_prefix!` in all the possibilities, please align them all to whatever flavour you prefer.

Reply:
I'm using sub because I need a regex to remove mirror url. I didn't find a way to use REG with delete_prefix.

Reply:
Do you want me to use sub for everything ?